db/migrate/20250718220533_change_subscriptions_id_to_bigint.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeSubscriptionsIdToBigint < ActiveRecord::Migration[7.1]
+  uses_departure! if Rails.env.staging? || Rails.env.production?
+
+  def up
+    change_column :subscriptions, :id, "bigint NOT NULL AUTO_INCREMENT"
+  end
+
+  def down
+    change_column :subscriptions, :id, "int NOT NULL AUTO_INCREMENT"
```

Comment:
Please remove this and replace it with `raise ActiveRecord::IrreversibleMigration` similar to how `20250515095633_change_user_agent_limit.rb` does it. We'd like this since we can't safely migrate to a smaller id (once there records with a `> int` id)